.github/workflows/scorecard-analysis.yml

```diff
@@ -3,9 +3,9 @@ on:
   push:
     # Only the default branch is supported.
     branches: [main, master]
-  pull_request:
+  #pull_request:
```

Comment:
Maybe put a TODO here saying we'll re-enable after implementing the local RepoClient